recipes/cose-c/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class CoseCStackConan(ConanFile):
+    name = "COSE-C"
```

Comment:
Use only lower-case for packages, even if the original package name is UPPER CASE or CamelCase